lms/djangoapps/instructor/management/commands/dump_grades.py

```diff
@@ -1,76 +0,0 @@
-#!/usr/bin/python
-"""
```

Comment:
Yes this management command dumps ALL grades for a course. It uses outdated, slow logic from the Legacy Instructor Dash. I believe this should not be in use now that you can generate grades from the Instructor Dash using a background Celery task.